dev-java/netbeans-nb/netbeans-nb-8.1.ebuild

```diff
@@ -9,8 +9,8 @@ HOMEPAGE="http://netbeans.org/"
 SLOT="8.1"
 SOURCE_URL="http://download.netbeans.org/netbeans/8.1/final/zip/netbeans-8.1-201510222201-src.zip"
 SRC_URI="${SOURCE_URL}
-	http://dev.gentoo.org/~fordfrog/distfiles/netbeans-8.1-build.xml.patch.bz2
-	http://dev.gentoo.org/~fordfrog/distfiles/netbeans-7.0.png"
+	https://dev.gentoo.org/~fordfrog/distfiles/netbeans-8.1-build.xml.patch.bz2
+	https://dev.gentoo.org/~fordfrog/distfiles/netbeans-7.0.png"
```

Comment:
@gentoo/java, @fordfrog, this one is 404. I'm going to commit the change since it doesn't regress it but you probably want to look into fixing that.

Reply:
We're about to drop 8.1 but this is in 8.2 as well.